pkg/reconciler/common/targetnamespace.go

```diff
@@ -56,7 +56,7 @@ func ReconcileTargetNamespace(ctx context.Context, labels map[string]string, ann
 		if namespace.Name == tektonComponent.GetSpec().GetTargetNamespace() && namespace.DeletionTimestamp == nil {
 			_targetNamespace := namespace.DeepCopy()
 			targetNamespace = _targetNamespace
-		} else {
+		} else if len(namespace.GetOwnerReferences()) > 0 {
```

Comment:
I manually created the ns, which resulted in entering here, causing a panic.

I understand that we should not panic under any circumstances.